src/utils.js

```diff
@@ -34,3 +34,23 @@ export function useInterval({ callback, delay, runOnStart }) {
     }
   }, [delay, runOnStart]);
 }
+
+export function isFeatureEnabled(feature) {
+  const experimentalFeatures = localStorage.getItem('dopplerExpermiental')
+    ? JSON.parse(localStorage.getItem('dopplerExpermiental'))
```

Comment:
What do you think about getting and parsing this item only one time?

Reply:
Now that you mention .. I should move this. Thanks!